src/modules/bifrost/test/CMakeLists.txt

```diff
@@ -13,14 +13,14 @@ get_filename_component(CAEN_FULL "${ESS_MODULE_DIR}/loki/configs/Loki.json" ABSO
 
 set(BifrostGeometryTest_INC 
     ../geometry/BifrostGeometry.h
-    ../geometry/Config.h
-    ../geometry/Calibration.h
+    ${ESS_SOURCE_DIR}/modules/caen/geometry/Config.h
+    ${ESS_SOURCE_DIR}/modules/caen/geometry/Calibration.h
```

Comment:
For references to files in the modules folder you should use ${ESS_MODULE_DIR} in stead.